lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenCompletionParticipant.java

```diff
@@ -513,6 +513,10 @@ private CompletionItem toTag(String name, MarkupContent description, ICompletion
 				CompletionItem item = toTextCompletionItem(request, "${" + property.getKey() + '}');
 				item.setDocumentation("Default Value: " + (property.getValue() != null ? property.getValue() : "unknown"));
 				item.setKind(CompletionItemKind.Property);
+				if (property.getKey().contains("env.")) {
+					// We don't want environment variables at the top of the completion proposals
+					item.setSortText("${z}");
```

Comment:
@mickaelistria this is a bit of a hack but it works. Maybe there's a more elegant way to do this.